.travis.yml

```diff
@@ -11,3 +11,6 @@ node_js:
 cache:
   directories:
     - node_modules
+after_script:
+  - npm install -g codeclimate-test-reporter
```

Comment:
you can ddd codeclimate-test-reporter to the devDependencies.
Example here:
https://github.com/airbnb/caravel/blob/master/caravel/assets/package.json#L95

And this is a sample PR where I've set up it to work:
https://github.com/airbnb/caravel/pull/1046

Only issue I couldn't fix was using the travis env variables. codeclimate-test-reporter doesn't see them